operations/accountingitems.md

```diff
@@ -370,7 +370,7 @@ For most amounts, precision of values depends on `TaxPrecision` of [Enterprise](
 
 ## Update accounting items
 
-Updates specified accounting items. You can use this operation to assign an accounting item to a different account or bill. Note this operation supports [Portfolio Access Tokens](../guidelines/multi-property.md).
+Updates specified accounting items. You can use this operation to assign an accounting item to a different account or bill. Note this operation supports [Portfolio Access Tokens](../concepts/multi-property.md).
```

Comment:
issue: We have links to multi-property everywhere in our annotations. Please open a follow-up ticket for us to update those links (although it's not a high priority as redirect will take care of it).